compiler/rustc_middle/src/traits/mod.rs

```diff
@@ -47,7 +47,8 @@ pub enum Reveal {
     /// impl. Concretely, that means that the following example will
     /// fail to compile:
     ///
-    /// ```
+    /// ```ignore HELP (is E0575 the right error for this?)
+    /// #![feature(specialization)]
```

Comment:
no, the comment originally forgot a variable:
```diff
diff --git a/compiler/rustc_middle/src/traits/mod.rs b/compiler/rustc_middle/src/traits/mod.rs
index da418eb1355..ffde1294ec6 100644
--- a/compiler/rustc_middle/src/traits/mod.rs
+++ b/compiler/rustc_middle/src/traits/mod.rs
@@ -47,7 +47,7 @@ pub enum Reveal {
     /// impl. Concretely, that means that the following example will
     /// fail to compile:
     ///
-    /// ```ignore HELP (is E0575 the right error for this?)
+    /// ```compile_fail,E0308
     /// #![feature(specialization)]
     /// trait Assoc {
     ///     type Output;
@@ -58,7 +58,7 @@ pub enum Reveal {
     /// }
     ///
     /// fn main() {
-    ///     let <() as Assoc>::Output = true;
+    ///     let x: <() as Assoc>::Output = true;
     /// }
     /// ```
     UserFacing,
```